crates/ruff_cli/src/printer.rs

```diff
@@ -178,9 +178,10 @@ impl Printer {
             SerializationFormat::Junit => {
                 JunitEmitter::default().emit(writer, &diagnostics.messages, &context)?;
             }
-            SerializationFormat::Text => {
+            SerializationFormat::Text | SerializationFormat::EcosystemCi => {
                 TextEmitter::default()
                     .with_show_fix_status(show_fix_status(self.autofix_level))
+                    // Currently only has an effect for ecosystem-ci
```

Comment:
What happens if you run `--fix --show-fixes` with the default text emitter? But I guess that would already have been a problem before.

Reply:
`--fix --show-fixes` currently has no effect over `--fix`, at least i didn't see any different output